web_src/js/components/PullRequestMergeForm.vue

```diff
@@ -10,8 +10,9 @@
       -d '{"context": "test/context", "description": "description", "state": "${state}", "target_url": "http://localhost"}'
   -->
   <div>
-    <!-- eslint-disable -->
-    <div v-if="mergeForm.hasPendingPullRequestMerge" v-html="mergeForm.hasPendingPullRequestMergeTip" class="ui info message"></div>
+    <div v-if="mergeForm.hasPendingPullRequestMerge" class="ui info message">
+      {{ mergeForm.hasPendingPullRequestMergeTip }}
+    </div>
```

Comment:
Template code indicates this is just a string, so `v-html` was completely unnecessary.

Reply:
No, it isn't a plain string.

Reply:
What is it? Template code maps it to a translation entry:

```
{{$hasPendingPullRequestMergeTip = $.i18n.Tr "repo.pulls.auto_merge_has_pending_schedule" .PendingPullRequestMerge.Doer.Name $createdPRMergeStr}}
```

```
pulls.auto_merge_has_pending_schedule = %[1]s scheduled this pull request to auto merge when all checks succeed %[2]s.
```

Reply:
No. It is HTML. `createdPRMergeStr`